### ISSUE TITLE
RTX 3090

### ISSUE THREAD

**Issue by joeorm** [ethereum-mining/ethminer#2255]
Hi,
I have an rtx 3090 running and it reports about 100 MH/s is that what is expected of it?
BR

**Comment by jean-m-cyr** [comment]
Don't know. I don't have one.

**Comment by RainbowMiner** [comment]
Yes, that's pretty much the value you would expect. The following is a snapshot of my monitoring insights on rbminer.net:
![image](https://user-images.githubusercontent.com/39437538/114266444-c02b4d80-99f6-11eb-82be-bf6b06dfb4ff.png)

**Comment by joeorm** [comment]
Thanks for the "RE" jean-m-cyr and RainbowMiner.

**Comment by funoverip** [comment]
With a bit of overclocking, I reach a stable 120MHs .

**Comment by Stoom** [comment]
My 3090 that has more airflow and a dedicated PSU can hit 120-130.  The 3090 inside the tower gets around 110-115 (Might be because of the programs running on that card too 🤷 )  Once I replaced the thermal pads on the card though everything has gone a lot better and no more thermal throttling.

**Comment by oralpast** [comment]
> With a bit of overclocking, I reach a stable 120MHs .

how to get a stable 120mh?

**Comment by funoverip** [comment]
> 

Here is my config (headless Ubuntu)

As root:

```
export DISPLAY=:0.0
xinit &
```
Then
```
nvidia-settings -c :0 -a '[gpu:0]/GPUGraphicsClockOffsetAllPerformanceLevels=-200' -a '[gpu:0]/GPUMemoryTransferRateOffsetAllPerformanceLevels=2600'
nvidia-settings -c :0 -a '[gpu:1]/GPUGraphicsClockOffsetAllPerformanceLevels=-280' -a '[gpu:1]/GPUMemoryTransferRateOffsetAllPerformanceLevels=2600'
nvidia-smi -i 0 -pl 310
nvidia-smi -i 1 -pl 320
```

Ethminer output:

```
 i 09:29:55 ethminer Job: da3629ae… eu1.ethermine.org [172.65.207.106:5555]
 m 09:29:56 ethminer 0:05 A19 241.91 Mh - cu0 121.67, cu1 120.24
 i 09:29:56 ethminer Job: 56d3133b… eu1.ethermine.org [172.65.207.106:5555]
```

**Comment by tryeth2021** [comment]
> > 
> 
> Here is my config (headless Ubuntu)
> 
> As root:
> 
> ```
> export DISPLAY=:0.0
> xinit &
> ```
> 
> Then
> 
> ```
> nvidia-settings -c :0 -a '[gpu:0]/GPUGraphicsClockOffsetAllPerformanceLevels=-200' -a '[gpu:0]/GPUMemoryTransferRateOffsetAllPerformanceLevels=2600'
> nvidia-settings -c :0 -a '[gpu:1]/GPUGraphicsClockOffsetAllPerformanceLevels=-280' -a '[gpu:1]/GPUMemoryTransferRateOffsetAllPerformanceLevels=2600'
> nvidia-smi -i 0 -pl 310
> nvidia-smi -i 1 -pl 320
> ```
> 
> Ethminer output:
> 
> ```
>  i 09:29:55 ethminer Job: da3629ae… eu1.ethermine.org [172.65.207.106:5555]
>  m 09:29:56 ethminer 0:05 A19 241.91 Mh - cu0 121.67, cu1 120.24
>  i 09:29:56 ethminer Job: 56d3133b… eu1.ethermine.org [172.65.207.106:5555]
> ```

thank you very much.I did it success.But why GPUMemoryTransferRateOffsetAllPerformanceLevels is so high in linux ?In windows,only set mclock with 1200 can get 120MH.
